### PR TITLE
Add a checklist for order management

### PR DESCRIPTION
**Pull request by rebekaburnett**
Also use anchors for the OpenAPI spec links - makes it easier to update them later.

### CODE REVIEW

File: vipps-order-management-api-checklist.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+ℹ️ Please use the new documentation:
+[Vipps Technical Documentation](https://vippsas.github.io/vipps-developer-docs/).
+
+<!-- END_COMMENT -->
+
+API version: 1.0.
+
+## Checklist
+
+- [ ] Integrate _all_ the [API endpoints][order-mgmt-api-reference-url]:
```
Comment (khenrichsen) [inline vipps-order-management-api-checklist.md:21]: Hm, this is not always necessary. It is fine for merchants to only use /receipts or /categories if they dont want both :)

File: vipps-order-management-api-checklist.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    - For examples of requests and responses, see the [Postman collection](tools/vipps-order-management-api-postman-collection.json) and [environment](https://github.com/vippsas/vipps-developers/blob/master/tools/vipps-api-global-postman-environment.json).
+- [ ] Send the [Vipps HTTP headers](https://vippsas.github.io/vipps-developer-docs/docs/vipps-developers/common-topics/http-headers)
+      in all API requests for better tracking and troubleshooting
+      (mandatory for partners and platforms, who must send these headers as part of the checklist approval):
+    - [ ] `Merchant-Serial-Number`
+    - [ ] `Vipps-System-Name`
+    - [ ] `Vipps-System-Version`
+    - [ ] `Vipps-System-Plugin-Name`
+    - [ ] `Vipps-System-Plugin-Version`
+- [ ] Follow the [orderId recommendations](vipps-ecom-api.md#orderid-recommendations).
```
Comment (khenrichsen) [inline vipps-order-management-api-checklist.md:36]: I guess this one is more relevant to have in the epayment chech-list?

File: vipps-order-management-api-quick-start.md [hunk 3]
```diff
@@ -65,21 +64,24 @@ Save the following files to your computer:
...
 
 The access token is valid for 1 hour in the test environment
 and 24 hours in the production environment.
 See the
-[API reference](https://vippsas.github.io/vipps-developer-docs/api/order-management)
+[API reference][order-mgmt-api-reference-url]
 for details about the calls.
 
-### Add receipt with order info and image
+### Add receipt with order info and image to an existing order
```
Comment (khenrichsen) [inline vipps-order-management-api-quick-start.md:77]: the order doesnt need to exist yet. It is possible to add the receipt first, then create the order afterwards. But, for simplicity i guess its fine :)

File: vipps-order-management-api-checklist.md (hunk 2 above)
Comment (rebekaburnett) [reply vipps-order-management-api-checklist.md]: updated

**Review by khenrichsen — APPROVED** [review]
Sweet!

**Pull request merged**